### DISCUSSION TITLE
Directus 9: How to update to newer release?

### DISCUSSION THREAD

**Discussion by madex1** [directus/directus#3726] [Archive]
Hello!
I'm using docker version of directus. Is it enough to change verison in my docker-compose.yml (i.e. rc-19 to rc-30), or i should run some migrations after that?

**Comment by benhaynes** [comment]
@WoLfulus ?

**Comment by WoLfulus** [comment]
Right now you should be good as there's no big changes happening, so Directus will auto migrate itself, but, keep in mind that we're on RC, so there's no guarantees it's going to work seamlessly on every release. 

After RC you should be good doing that while upgrading `patch` releases - any minor and major version upgrades you should take a look at the release notes first.

Note that .29 and .30 isn't yet available as docker images. I'm waiting a bit to see what happens to https://github.com/mapbox/node-sqlite3/issues/1424 before pinning the version.
